modules/serverless-broker-backing-channel-default.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+//  * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-broker-backing-channel-default_{context}"]
+= Configuring the default broker backing channel
+
+If you are using a channel-based broker, you can set the default backing channel type for the broker to either `InMemoryChannel` or `KafkaChannel`.
+
+.Prerequisites
+
+* You have administrator permissions on {product-title}.
+* You have installed the {ServerlessOperatorName} and Knative Eventing on your cluster.
+* If you want to use Kafka channels as the default backing channel type, you must also install the `KnativeKafka` CR on your cluster.
+
+.Procedure
+
+* Modify the `KnativeEventing` custom resource (CR) to add configuration details for the `config-br-default-channel` config map:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  config: <1>
+    config-br-default-channel:
+      channel-template-spec: |
+        apiVersion: messaging.knative.dev/v1beta1
+        kind: KafkaChannel <2>
+        spec:
+          numPartitions: 6 <3>
+          replicationFactor: 1 <4>
```

Comment:
nit: this can be whatever but I would prefer something safer as an example:
```suggestion
          replicationFactor: 3 <4>
```

Reply:
`3` is also what we have (for the broker) as default, b/c its more real, than just `1`